src/main/java/ru/yandex/practicum/filmorate/controllers/FilmController.java

```diff
@@ -0,0 +1,40 @@
+package ru.yandex.practicum.filmorate.controllers;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.*;
+import ru.yandex.practicum.filmorate.model.Film;
+import ru.yandex.practicum.filmorate.service.FilmService;
+import javax.validation.Valid;
+import java.time.LocalDate;
+import java.util.List;
+@Slf4j
+@RestController
+@RequestMapping("/films")
+public class FilmController {
+
+private final FilmService filmService;
+
+    public FilmController(FilmService filmService) {
+        this.filmService = filmService;
+    }
+
+    @PostMapping
+    public Film create(@Valid @RequestBody Film film){
```

Comment:
Если отправить такой POST запрос: http://localhost:8080/films
{
	"name":"FilmName",
	"description":"descriptionItem",
	"duration": 22
}
То падает
java.lang.NullPointerException: Cannot invoke "java.time.LocalDate.isBefore(java.time.chrono.ChronoLocalDate)" because the return value of "ru.yandex.practicum.filmorate.model.Film.getReleaseDate()" is null
	at ru.yandex.practicum.filmorate.controllers.FilmController.create(FilmController.java:23) ~[classes/:na]